modules/assim.sequential/DESCRIPTION

```diff
@@ -28,7 +29,9 @@ Imports:
     PEcAn.uncertainty,
     PEcAn.workflow,
     purrr,
+    raster,
     rlang,
+    sf,
     stringr
 Suggests:
     corrplot,
```

Comment:
```suggestion
    corrplot,
    exactextractr,
```
I'd prefer this package be in Suggests since the new function is optional and we're restricted to <20 Imports